### PR TITLE
Updates to config blueprints

### PR DESCRIPTION
**Pull request by yamini**
Better support for low record count or high dimensionality. Added model names to all configs.

### CODE REVIEW

File: config_templates/gretel/synthetics/high-accuracy.yml [hunk 1]
```diff
@@ -11,12 +12,17 @@
...
       params:
         epochs: 100
         vocab_size: 0
         learning_rate: .001
-        rnn_units: 256
\ No newline at end of file
+        rnn_units: 256
+        reset_states: false
+        # Gretel privacy filtering
+        privacy_filters:
```
Comment (johntmyers) [inline config_templates/gretel/synthetics/high-accuracy.yml:26]: @yamini this might be one level in too far

Comment (yamini) [reply config_templates/gretel/synthetics/high-accuracy.yml:26]: Thanks for catching that.
